src/components/genre-select/genre-select.tsx

```diff
@@ -0,0 +1,18 @@
+import { Select, SelectOptions } from "../../ui";
+
+type Props = {
+    options: SelectOptions;
+    selected: string;
+    onSelect: (name: string) => void;
+};
+export const GenreSelect = ({ options, onSelect, selected }: Props) => {
```

Comment:
On some projects something like this is used:

```suggestion
export const GenreSelect: FC<Props> = ({ options, onSelect, selected }) => {
```